### PR TITLE
Phone number: Correct the format

### PR DESCRIPTION
**Pull request by iHiD**
Context: https://forum.exercism.org/t/nanp-phone-numbers/7179/1

This shouldn't change any tests, so I think it's pretty safe to merge.

### CODE REVIEW

**Review by IsaacG — APPROVED** [review]

**Review by kotp — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
> This shouldn't change any tests, so I think it's pretty safe to merge.

True, but most of the tests use the old format, so that might be a bit weird.

File: exercises/phone-number/description.md [hunk 1]
```diff
@@ -11,12 +11,14 @@ The first three digits of the local number represent the *exchange code*, follow
 The format is usually represented as
 
 ```text
-(NXX)-NXX-XXXX
+NXX NXX-XXXX
```
Comment (kotp) [inline exercises/phone-number/description.md:14]: We might show both that are often used... 

The tests are not going to differentiate in a way that errors, but will present it as a cleaned version.

Here we are showing what is "usually" represented, and I might think that 

```text
(NXX) NXX-XXXX
```

And

```text
(NXX)NXX-XXXX
```

are pretty evenly tied in the usual use.

I totally agree with using either with parenthesis or or dash to separate the area code from either or both sides of that, never both at the same time.

**Review by kotp — APPROVED** [review]

File: exercises/phone-number/description.md (hunk 1 above)
Comment (IsaacG) [reply exercises/phone-number/description.md:14]: We could also change "usually" to "sometimes" and not worry about what is commonly used where :smile:

**Comment by iHiD** [comment]
In response to the comments in general, I think the point is that the tests provide different formats but really the input format is irrelevant - what matters is that it gets normalised. So I think we should make the description actively generic, rather than more specific. If that makes sense?

**Review by BethanyG — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
Totally agree

**Review by kytrinyx — APPROVED** [review]

**Pull request merged**